client/src/components/CredentialsInputBox/CredentialsInputBox.js

```diff
@@ -0,0 +1,18 @@
+import React from "react";
+import "./CredentialsInputBox.css";
+
+export default function CredentialsInputBox() {
+  return (
+    <div className="cred-layout">
+      <b className="cred-input-header">Email</b>
+      <input
+        className="cred-inputbox"
+        type="text"
+        name="name"
+        placeholder="utorid@utoronto.ca"
+      />
+      <b className="cred-input-header">Password</b>
+      <input className="cred-inputbox" type="password" name="name" />
```

Comment:
I would also include a placeholder for the password so it stays consistent with the email placeholder

Reply:
Fixed (Nestor):  added "your password" placeholder.